lib/streamlit/elements/utils.py

```diff
@@ -26,7 +26,7 @@
     from streamlit.type_util import DataFrameCompatible
 
 
-def clean_text(text: object) -> str:
+def clean_text(text: type_util.SupportsStr) -> str:
```

Comment:
Two small things:

* Since we're already importing `DataFrameCompatible` from `streamlit.type_util` above, it'd be slightly preferable to include `SupportsStr` in that import line. Alternatively, we could not import `DataFrameCompatible` and access it through `type_util` since that's also being imported above. Either of the two seem equally okay to me.
* Additionally, is there any significance in the actual type being used here, but being wrapped in quotes in `alert.py`?

Reply:
Good catch of the untidy imports. Have you guys considered a linting tool for imports, like `isort`? Having a strictly enforced import pattern would reduce the amount of attention you would need to divert to imports during review, leaving more time for the salient parts of PRs. Worth considering, IMO.

As for quoted types:
Prior to python 3.7, types might need to be quoted in order to not sabotage the code at runtime. This is because the type statements are eagerly executed. This changes in python 3.7 because you can [enable postponed evaluation](https://peps.python.org/pep-0563/). I haven't done so, because Streamlit still implicitly supports python 3.6.

A typical case for when you need to wrap types in strings is when you import them conditionally under `if TYPE_CHECKING:`. This implies they're not available at runtime, and hence, would lead to an exception. So, whether or not I use quotes depends on the method of import in this case.

Reply:
re: using `isort`. It's definitely something we've considered, but so far it has been sufficiently non-urgent to get pushed off repeatedly up until this point. I'll add it to my personal TODO list, though, given that I do think that the small amount of time invested to just install/use it will quickly pay for itself.